tools/docs/manager.ts

```diff
@@ -1,4 +1,6 @@
 import type { RenovateConfig } from '../../lib/config/types';
```

Comment:
Did you remove the code that created the old list-style layout already? It's an easy thing to forget, maybe?

Reply:
Yes, that has been removed with 
https://github.com/renovatebot/renovate/pull/23503/files#diff-a043b8e3caecfea79270d50759b258a042174acd67328b0d8371a2e6516ceec7L129-L131